packages/fxa-payments-server/src/lib/errors.test.tsx

```diff
@@ -1,16 +1,28 @@
 import '@testing-library/jest-dom/extend-expect';
 import { getErrorMessage, BASIC_ERROR, PAYMENT_ERROR_1 } from './errors';
 
+it('returns the basic error text if undefined error type', () => {
+  expect(getErrorMessage(undefined)).toEqual(BASIC_ERROR);
+});
+
 it('returns the basic error text if not predefined error type', () => {
-  expect(getErrorMessage('NON_PREDEFINED_ERROR')).toEqual(BASIC_ERROR);
+  expect(getErrorMessage({ code: 'NON_PREDEFINED_ERROR' })).toEqual(
+    BASIC_ERROR
+  );
```

Comment:
I know you didn't write this test, but I don't see this `'NON_PREDEFINED_ERROR'` used anywhere in the source code. How is it different from the `undefined` case above? Can we remove it?

Reply:
It's different because `undefined` handles the case where nothing is passed to getErrorMessage. This handles the case where a mystery error with a code that the system isn't coded for is passed to getErrorMessage.